libnymea/jsonrpc/jsonhandler.cpp

```diff
@@ -359,6 +359,10 @@ QVariant JsonHandler::pack(const QMetaObject &metaObject, const void *value) con
                         foreach (const EventTypeId &entry, propertyValue.value<QList<EventTypeId>>()) {
                             list << entry;
                         }
+                    } else if (propertyTypeName == "QList<QDateTime>") {
+                        foreach (const QDateTime &timestamp, propertyValue.value<QList<QDateTime>>()) {
+                            list << timestamp.toTime_t();
```

Comment:
QDoc says: `This function is obsolete. It is provided to keep old source code working. We strongly advise against using it in new code.`

Maybe use `toMSecsSinceEpoch() / 1000`

Reply:
fixed.